docs/source/kubernetes/minikube/step-zero-minikube.md

```diff
@@ -0,0 +1,46 @@
+(minikube-k8s)
+
+# Kubernetes on minikube (for development only)
+
+[minikube](minikube) is a implementation of Kubernetes as a local cluster primarily target to the development of applications.
+
+```{important}
+The Zero to JupyterHub guide assumes you're using a managed Kubernetes service with one of the main cloud platforms and **[minikube](minikube) is not officially supported**. You may be able to get help on the [Jupyter community forum](https://discourse.jupyter.org/c/jupyterhub/10).
+```
+
+## Kubernetes cluster requirements
+
+All the requirements are implemented in minikube >= v1.31.2:
+
+- [Dynamic Volume Provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/) for persistent storage
+- [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) or [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for managing external access to JupyterHub
+
+## minikube installation
+
+Follow the installation steps in the [minikube's "Get Started!" page](https://minikube.sigs.k8s.io/docs/start/).
+
+## Kubernetes cluster creation
+
+From a terminal, run
+
+```bash
+minikube start \
+--kubernetes-version v1.26.1 \
```

Comment:
Not using a pre-defined version makes this docs more long lived, we could omit it entirely and rely on the default being "stable", but its kind of nice to explicitly pass this as well so that its known that this can be configured at this point in time also.

```suggestion
--kubernetes-version stable \
```